index.js

```diff
@@ -11,6 +11,7 @@ function CompressionPlugin(options) {
 	options = options || {};
 	this.asset = options.asset || "[path].gz[query]";
 	this.algorithm = options.algorithm || "gzip";
+	this.changeNewFileName = options.changeNewFileName || false;
```

Comment:
@ronen-e Ooops..:stuck_out_tongue forget what I said then 🙃  Just change to`filename` + docs and good to go imho